src/main/java/uk/gov/pay/api/model/search/card/RefundForSearchRefundsResult.java

```diff
@@ -95,6 +104,13 @@ public RefundLinksForSearch getLinks() {
         return links;
     }
 
+    @ApiModelProperty(hidden = true, dataType = "uk.gov.pay.api.model.SettlementSummary")
+    @JsonProperty("settlement_summary")
+    @Schema(hidden = true)
```

Comment:
I think the annotations on either the getter or the field can be removed?

Reply:
I agree, it is strange way but I just followed the existing way of annotating fields/getters